angular-mocks/index.d.ts

```diff
@@ -301,6 +301,25 @@ declare module 'angular' {
     whenPUT(url: string | RegExp | ((url: string) => boolean), data?: string | RegExp | Object | ((data: string) => boolean), headers?: Object | ((object: Object) => boolean), keys?: Object[]): mock.IRequestHandler;
   }
 
+  ///////////////////////////////////////////////////////////////////////////
+  // AnimateService
+  // see https://docs.angularjs.org/api/ngMock/service/$animate
+  ///////////////////////////////////////////////////////////////////////////
+  module animate {
+    interface IAnimateService {
+
+      /**
+       * This method will close all pending animations (both Javascript and CSS) and it will also flush any remaining animation frames and/or callbacks.
+       */
+      closeAndFlush();
```

Comment:
What is `closeAndFlush`'s return type? I guess it should be void, right? Right now it's implicitly any, which causes CI to fail.